app/services/members-utils.js

```diff
@@ -121,8 +121,8 @@ export default class MembersUtilsService extends Service {
             settingsParam.append('portalPrices', encodeURIComponent(portalPlans));
         }
 
-        if (portalProducts && this.feature.get('multipleProducts')) {
-            settingsParam.append('portalProducts', encodeURIComponent(portalProducts));
+        if (portalTiers && this.feature.get('multipleProducts')) {
```

Comment:
Not sure about this one

Reply:
Hm yeah this one might not work right now, because we use the setting called `portalProducts` in Portal preview to show all the selected products in Portal, so this won't be just a variable change. I think it's ok to keep this one as `portalProducts` for now ?

Reply:
Updated!